specs/validator/0_beacon-chain-validator.md

```diff
@@ -198,7 +198,7 @@ Specifically a validator should:
 * Call `get_committee_assignment(state, next_epoch, validator_index)` when checking for next epoch assignments.
 * Join the pubsub topic -- `committee_index{committee_index % ATTESTATION_SUBNET_COUNT}_beacon_attestation`.
     * If any current peers are subscribed to the topic, the validator simply sends `subscribe` messages for the new topic.
```

Comment:
Considering changing the language here to "sufficient" number of peers, otherwise go find some more. "Any" implies that 1 is enough.. which it might be, but a validator might also want to go find some more to better disseminate their attestation.